LibreNMS/OS/AviatWtm.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * AviatWtm.php
+ *
+ * Aviat WTM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Josh Baird
+ * @author     Josh Baird<joshbaird@gmail.com>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessFrequencyDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRateDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSnrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessPowerDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRssiDiscovery;
+use LibreNMS\OS;
+
+class AviatWtm extends OS implements
+    WirelessFrequencyDiscovery,
+    WirelessRateDiscovery,
+    WirelessRssiDiscovery,
+    WirelessSnrDiscovery,
+    WirelessPowerDiscovery
+{
+  /**
+	 * Discover wireless tx or rx power. This is in dBm. Type is power.
+	 * Returns an array of LibreNMS\Device\Sensor objects that have been discovered
+	 *
+	 * @return array
+   */
+
+  public function discoverWirelessFrequency()
+  {
+      $carrier1_oid = '.1.3.6.1.4.1.2509.9.5.2.1.1.1.59';
+      $carrier2_oid = '.1.3.6.1.4.1.2509.9.5.2.1.1.1.60';
+      # The SNMP agent on WTM devices will choke on 15+ rapid requests.
+      # A "sleep" helps LibreNMS cope with this current issue
+      sleep(5);
```

Comment:
These sleep can have a very detrimental effect on LibreNMS.  You'll need another solution.
Try setting nobulk or snmp_max_oid in the os def.